### DISCUSSION TITLE
Georam issue

### DISCUSSION THREAD

**Discussion by C64Retro** [frntc/Sidekick64#93] [Q&A]
Hi, I have built up a couple of sidekick64's and they seem to be both working expected with most functions. However, I seem to have an issue with the georam. I am running latest build 0.51d. I have used a few test programs and georam passes so all ok there, but the problem is no matter what ram setting I have for georam 0.5-4MB in settings (I have tried all), it seems to be detected as 4MB with testing program (georam64.prg). The TFT display however does show correct ram setting which was set.

When testing the Hudson Hawk georam game it loads 1st time (but this time TFT will show 4MB no matter what ram I have set in settings), but once resetting or powering off the game it can not be loaded again and the file size of the game is changed from 512KB to 4MB. Changing the game file size back to 512KB will make it work for the 1st time once again. Thanks in advance I hope someone can offer some suggestions.

**Comment by frntc** [comment]
Do you have a link to georam64.prg?

The problem you describe with Hudson Hawk is something that I think I've fixed in the past... need to check!

**Reply by C64Retro** [reply]
The programs I have tried are: georam64.prg (attached) and GeoRAMChecker.d64 which has a georam-checker.prg. When running these no matter georam is set to with sidekick, the programs will show 4MB. The sidekick oled however will show the correct georam setting.
With Hudson Hawk the file size is 512KB but when run on sidekick the file size gets overwritten to 4096KB and then can't run it again. Have to copy over again to run once again with same issue. I have built 2 sidekicks with their own Pi3A+ and same issues with both builds.
I have attached zip of the files.

Thanks fr your assistance.
[georam.zip](https://github.com/frntc/Sidekick64/files/12837432/georam.zip)

**Reply by frntc** [reply]
there was indeed a bug, thanks for reporting. I plan to release an update in the next days, it'll include a fix.

**Comment by C64Retro** [comment]
Thanks for your continued work on this project. It's really appreciated.
Regards Brad
